### PR TITLE
added custom logger to wrap console with UI for controlling configs are runtime

### PR DESCRIPTION
**Pull request by kamikaz1k**
image

### CODE REVIEW

File: src/logger.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```
Comment (kamikaz1k) [inline src/logger.ts:1]: the linter was really annoying for this module, so I disabled it

File: src/index.ts [hunk 2]
```diff
@@ -21,3 +21,47 @@ const config: Phaser.Types.Core.GameConfig = {
...
 console.log('### Game', game);
+
+import {
+  setLoggingConfig,
+  getLoggingConfig,
+  LOGGING_KEYS_ALL,
+  LOGGING_KEYS,
+} from './logger';
+
+(function loadDebugLoggerConfigurations() {
```
Comment (kamikaz1k) [inline src/index.ts:32]: tbh, no real reason for IIFE....

File: src/logger.ts (hunk 1 above)
Comment (nafeu) [reply src/logger.ts:1]: That's fair man, you are welcome to use eslint-disable literally wherever you think it's doing more harm then good

**Review by nafeu — APPROVED** [review]
This is great man thank you! You are good to merge

**Pull request merged**